Usage of deprecated Qt methods

See the following build log:
```
/tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp: In member function 'void MainWindow::_update()':
/tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp:200:22: error: 'void QTime::start()' is deprecated: Use QElapsedTimer instead [-Werror=deprecated-declarations]
  200 |     _frameTime.start();
      |                      ^
In file included from /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/merge/deploy/external-libraries/2020-07-20/linux-rhel7-x86_64/gcc-9.3.0/qt-5.14.2-ts7jj3x4eq/include/QtCore/QTime:1,
                 from /tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.h:53,
                 from /tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp:41:
/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/merge/deploy/external-libraries/2020-07-20/linux-rhel7-x86_64/gcc-9.3.0/qt-5.14.2-ts7jj3x4eq/include/QtCore/qdatetime.h:228:55: note: declared here
  228 |     QT_DEPRECATED_X("Use QElapsedTimer instead") void start();
      |                                                       ^~~~~
/tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp: In member function 'void MainWindow::_regulateFrameRate()':
/tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp:421:44: error: 'int QTime::elapsed() const' is deprecated: Use QElapsedTimer instead [-Werror=deprecated-declarations]
  421 |     const int elapsed = _frameTime.elapsed();
      |                                            ^
In file included from /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/merge/deploy/external-libraries/2020-07-20/linux-rhel7-x86_64/gcc-9.3.0/qt-5.14.2-ts7jj3x4eq/include/QtCore/QTime:1,
                 from /tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.h:53,
                 from /tmp/matwolf/spack-stage-deflect-1.0.3-7yb4fmqolpo7mzmghhohk7unxz5l6oks/spack-src/apps/DesktopStreamer/MainWindow.cpp:41:
/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/merge/deploy/external-libraries/2020-07-20/linux-rhel7-x86_64/gcc-9.3.0/qt-5.14.2-ts7jj3x4eq/include/QtCore/qdatetime.h:230:54: note: declared here
  230 |     QT_DEPRECATED_X("Use QElapsedTimer instead") int elapsed() const;
      |                                                      ^~~~~~~
cc1plus: all warnings being treated as errors
```

Would be nice if we could update the API to follow the latest non-deprecated Qt.

--- a/apps/DesktopStreamer/MainWindow.h
+++ b/apps/DesktopStreamer/MainWindow.h
@@ -49,8 +49,8 @@
 
 #include <deflect/types.h>
 
-#include <QMainWindow>
 #include <QElapsedTimer>
+#include <QMainWindow>
 #include <QTimer>
 #include <map>
 #include <memory>

--- a/apps/DesktopStreamer/MainWindow.h
+++ b/apps/DesktopStreamer/MainWindow.h
@@ -50,7 +50,7 @@
 #include <deflect/types.h>
 
 #include <QMainWindow>
-#include <QTime>
+#include <QElapsedTimer>
 #include <QTimer>
 #include <map>
 #include <memory>
@@ -93,7 +93,7 @@ private:
     uint32_t _streamID;
 
     QTimer _updateTimer;
-    QTime _frameTime;
+    QElapsedTimer _frameTime;
     float _averageUpdate;
 
 #ifdef __APPLE__

